refs.bib

```diff
@@ -0,0 +1,8 @@
+@article{Doe2016,
+  author = {Doe, J and Smith, R},
+  journal = {Title of the journal},
+  pages = {10--30},
+  title = {Title of the paper},
```

Comment:
This should have double curly braces, otherwise uppercases other than first letter are transformed to lowercase when creating the reference list

So always use `title = {{Title of the Paper}}`